python/orca/tutorial/pytorch/FashionMNIST/orca-tutorial-k8s-client.yaml

```diff
@@ -10,66 +10,56 @@ spec:
       containers:
       - name: spark-k8s-client
-        image: intelanalytics/bigdl-k8s:2.1.0
+        image: intelanalytics/bigdl-k8s:latest
         imagePullPolicy: IfNotPresent
         command: ["/bin/sh","-c"]
         args: ["
                 export RUNTIME_DRIVER_HOST=$( hostname -I | awk '{print $1}' );
                 ${SPARK_HOME}/bin/spark-submit \
                 --master ${RUNTIME_SPARK_MASTER} \
                 --deploy-mode ${SPARK_MODE} \
+                --name orca-k8s-client-tutorial \
                 --conf spark.driver.host=${RUNTIME_DRIVER_HOST} \
-                --conf spark.driver.port=${RUNTIME_DRIVER_PORT} \
-                --conf spark.kubernetes.authenticate.driver.serviceAccountName=${RUNTIME_K8S_SERVICE_ACCOUNT} \
-                --name orca-pytorch-tutorial \
                 --conf spark.kubernetes.container.image=${RUNTIME_K8S_SPARK_IMAGE} \
                 --conf spark.kubernetes.driver.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName=nfsvolumeclaim \
                 --conf spark.kubernetes.driver.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path=/bigdl/nfsdata/ \
                 --conf spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.options.claimName=nfsvolumeclaim \
                 --conf spark.kubernetes.executor.volumes.persistentVolumeClaim.nfsvolumeclaim.mount.path=/bigdl/nfsdata/ \
-                --archives file:///bigdl/nfsdata/environment.tar.gz#orca_env \
-                --conf spark.pyspark.driver.python=/bigdl/nfsdata/orca_env/bin/python \
-                --conf spark.pyspark.python=orca_env/bin/python \
-                --conf spark.executorEnv.PYTHONHOME=orca_env \
-                --conf spark.kubernetes.file.upload.path=/bigdl/nfsdata/ \
+                --archives file:///bigdl/nfsdata/environment.tar.gz#environment \
+                --conf spark.pyspark.driver.python=/bigdl/nfsdata/spark_submit_environment/bin/python \
+                --conf spark.pyspark.python=environment/bin/python \
                 --num-executors 2 \
-                --executor-cores 16 \
-                --executor-memory 50g \
-                --total-executor-cores 32 \
-                --driver-cores 4 \
-                --driver-memory 50g \
+                --executor-cores 4 \
+                --executor-memory 2g \
+                --total-executor-cores 8 \
+                --driver-cores 2 \
+                --driver-memory 2g \
                 --properties-file ${BIGDL_HOME}/conf/spark-bigdl.conf \
-                --py-files local://${BIGDL_HOME}/python/bigdl-spark_${SPARK_VERSION}-${BIGDL_VERSION}-python-api.zip,local:///bigdl/nfsdata/train.py,local:///bigdl/nfsdata/model.py \
-                --conf spark.driver.extraJavaOptions=-Dderby.stream.error.file=/tmp \
+                --py-files ${BIGDL_HOME}/python/bigdl-spark_${SPARK_VERSION}-${BIGDL_VERSION}-python-api.zip,/bigdl/nfsdata/kai/train.py,/bigdl/nfsdata/kai/model.py \
```

Comment:
adding local:// will cause bigdl and model modules not found. Should be the same as: https://github.com/intel-analytics/BigDL/pull/7532#discussion_r1105708229